api/areaProfile.go

```diff
@@ -109,3 +111,94 @@ func (s *Server) currentAreaProfile(c *gin.Context) {
 		"neighbor":             metric,
 	})
 }
+
+func (s *Server) placeProfile(c *gin.Context) {
+	accountNumber := c.GetString("requester")
+
+	profile, err := s.mongoStore.GetProfile(accountNumber)
+	if err != nil {
+		abortWithEncoding(c, http.StatusInternalServerError, errorInternalServer, err)
+		return
+	}
+
+	poiID, err := primitive.ObjectIDFromHex(c.Param("poiID"))
+	if err != nil {
+		abortWithEncoding(c, http.StatusBadRequest, errorInvalidParameters, fmt.Errorf("invalid POI ID"))
+		return
+	}
+
+	// resources
+	var params struct {
+		Language     string `form:"lang"`
+		AllReosurces bool   `form:"all_resources"`
+	}
+	if err := c.Bind(&params); err != nil {
+		abortWithEncoding(c, http.StatusBadRequest, errorInvalidParameters, err)
+		return
+	}
+
+	if "" == params.Language {
+		params.Language = "en"
+	}
+
+	var profilePoi schema.ProfilePOI
+	for _, p := range profile.PointsOfInterest {
+		if p.ID == poiID {
+			profilePoi = p
+			break
+		}
+	}
```

Comment:
If `poiID` is not in user's profile,  we might need to return an error.

Reply:
ok